src/Import/src/FMI3/fmi3_import_capi.c

```diff
@@ -104,14 +116,19 @@ jm_status_enu_t fmi3_import_create_dllfmu(fmi3_import_t* fmu, fmi3_fmu_kind_enu_
                 logMessageFinal, fmuKind);
     }
 
+	if (fmu->capi) {
+		/* Replace the CAPI options with the import ones */
```

Comment:
I think the indentation needs some adjustment.

Reply:
Fixed.